build.fsx

```diff
@@ -84,6 +103,28 @@ Target.create "RunClient" (fun _ ->
     yarn webpackCommand
 )
 
+Target.create "RunUiTests" (fun _ ->
+    runDotNet "build" clientTests
+
+    let server = serverWatcher()
+    let client = clientWatcher false
+
+    let serverTask = 
+        [ server; client ] 
+        |> Async.Parallel 
+        |> Async.StartAsTask
+    sleep 15000 |> Async.RunSynchronously
+
+    runDotNet clientTestExecutables clientTests
```

Comment:
Well that produces `dotnet bin/Debug/netcoreapp3.1/Client.UiTests.dll ./tests/Client.UiTests` whereas the first parameter is supposed to be a command like "build" or "restore". So I suggest creating another function or refactoring.

Reply:
that is actually how I start this from console

Reply:
Sure, I mean because `bin/Debug/netcoreapp3.1/Client.UiTests.dll` is a path rather than "command", I suggest adding another function to run exes:
```fsharp
let runDotNet cmd workingDir =
    let result =
        DotNet.exec (DotNet.Options.withWorkingDirectory workingDir) cmd ""
    if result.ExitCode <> 0 then failwithf "'dotnet %s' failed in %s" cmd workingDir

let runExe path = 
    DotNet.exec (DotNet.Options.withWorkingDirectory ".") path "" |> ignore
```
and also
```
let clientTestsPath = Path.getFullName "./tests/Client.UiTests"
let clientTestExecutables = Path.combine clientTestsPath "bin/Debug/netcoreapp3.1/Client.UiTests.dll"
```
and then, in the target code
```
runDotNet "build" clientTestsPath
...
runExe clientTestExecutables
```
Would be just somewhat more explicit.

Reply:
done :-)

Reply:
Thanks!